packages/kmod-6.1-nvidia/open-gpu-depmod.conf.in

```diff
@@ -0,0 +1,2 @@
+# Force the proprietary driver to be excluded since the open-gpu path will be preferred
```

Comment:
I'm not sure why these `depmod` files are required if only one set of drivers is ever copied into the modules tree, as seems to be the goal of the current implementation.